swan-lake/featured-scenarios/manage-data-persistence-with-bal-persist.md

```diff
@@ -0,0 +1,502 @@
+---
+layout: ballerina-building-a-data-service-with-bal-persist-left-nav-pages-swanlake
+title: Manage data persistence with bal persist
+description: Describe how to model your application data and use the tool to generate client APIs to access the data store.
+keywords: ballerina, data service, data store, database, in-memory, google sheets, REST, API
+permalink: /learn/manage_data_persistence-with-bal-persist/
+active: manage_data_persistence-with-bal-persist
+intro: This guide helps you understand the basics of the bal persist, which allows you to manage data persistence easily. This same data service is also written using the Ballerina SQL connectors, which required you to write SQL queries to perform CRUD operations against the DB servers. With the Bal persist feature, you only need to write the `Data Model`. Based on the model, the client object and record types are generated to interact with the data store.
+
+redirect_from:
+- /learn/manage-data-persistence-with-bal-persist/
+- /learn/manage-data-persistence-with-bal-persist
+- /learn/manage-data-persistence-with-bal-persist
+- /learn/getting-started/manage-data-persistence-with-bal-persist/
+- /learn/getting-started/manage_data_persistence-with-bal-persist
+---
+
+## Set up the prerequisites
+
+To complete this tutorial, you need:
+
+1. [Ballerina 2201.6.0 (Swan Lake)](/learn/install-ballerina/set-up-ballerina/) or greater
+2. A text editor
+>**Tip:** Preferably, <a href="https://code.visualstudio.com/" target="_blank">Visual Studio Code</a> with the
+<a href="https://wso2.com/ballerina/vscode/docs/get-started/install-the-extension/" target="_blank">Ballerina extension</a> installed.
+3. A command terminal
+
+## Understand the implementation
+
+This tutorial describes how to interact with the data store and perform operations against it using bal persist. This will cover the basic use case of creating, reading, updating, and deleting records on a data store in an organization. It also elaborates on how you can create an HTTP RESTful API using Ballerina that can be used to perform basic CRUD operations on the data store.
```

Comment:
What is the correct term to refer to the feature ? `bal persist` or `Ballerina persist` (Can see the latter in below `Info` block.) We need to be consistent.